vignettes/cookbook.Rmd

```diff
@@ -341,7 +345,8 @@ df <- tibble(
 p <- ggplot(data = df,
             mapping = aes(
               x = reorder(RACE, -RESPONSE), # Order race by decreasing response rate
-              y = RESPONSE, fill = YEAR, label = LABEL
+              y = RESPONSE, fill = YEAR, 
+              label = LABEL # note that this label definition is implicitly referenced by `geom_text()` (below)
```

Comment:
Maybe this should be moved into `geom_text`'s `aes`, since that's the only geom that uses it?

Reply:
Makes sense, I can adjust.